apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/GenesisImporter.java

```diff
@@ -144,131 +264,234 @@ public Block newGenesisBlock() {
     }
 
     @Transactional
-    public void importGenesisJson(boolean loadOnlyPublicKeys) {
-        long start = System.currentTimeMillis();
-        if (this.balances == null || this.publicKeys == null) {
-            loadBalancesAccountsComputeDigest();
-        }
-        blockchainConfigUpdater = CDI.current().select(BlockchainConfigUpdater.class).get();
-        blockchainConfigUpdater.reset();
-        TransactionalDataSource dataSource = databaseManager.getDataSource();
+    public void importGenesisJson(final boolean loadOnlyPublicKeys) {
+        final long start = System.currentTimeMillis();
+
+        this.blockchainConfigUpdater = CDI.current().select(BlockchainConfigUpdater.class).get();
```

Comment:
We don't have to use the CDI selection in the refactored code.
There is already an injected instance of BlockchainConfigUpdater.

Reply:
I agree, but I haven't written this line of code from scratch. I only added `this` to avoid ambiguity.
![изображение](https://user-images.githubusercontent.com/56535960/67220938-577f9800-f433-11e9-866b-cd0a6f696812.png)

Reply:
fixed